setup.cfg

```diff
@@ -0,0 +1,60 @@
+[metadata]
+name = charmed-openstack-upgrader
+# See setup.py version generator
+url = https://github.com/esunar/charmed-openstack-upgrader
+description = Application for upgrading Charmed Openstack
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8; variant=GFM
+author = Canonical DevOps Centres
+license = GPLv3
+license_files = LICENSE
+classifiers =
+   Development Status :: 2 - Pre-Alpha
+   Environment :: Plugins
+   Intended Audience :: Developers
+   Intended Audience :: System Administrators
+   License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+   Operating System :: OS Independent
+   Topic :: System :: Installation/Setup
+   Topic :: Utilities
+   Programming Language :: Python :: 3
+   Programming Language :: Python :: 3.10
+
+[options]
+include_package_data = True
+python_requires = >=3.10
+packages = find:
+install_requires =
+    oslo.config
+    juju<3.0
+[options.extras_require]
+lint =
```

Comment:
Are we moving the requirements packages to `pyproject.toml` instead of `setup.cfg`?

Reply:
Personally do not know, if it's works I'm +1 for it. @esunar Did you check if it possible?